exchange/howto/run-seednode.adoc

```diff
@@ -4,108 +4,308 @@
 
 == Introduction
```

Comment:
AsciiDoc automatically makes the first line into a subtitle of sorts, so I would suggest replacing lines 5-13 with the following. It includes minor edits in spacing, commas, etc (no material changes).
```
Bisq seed nodes serve as static gateways for Bisq clients to get connected to Bisq's dynamic P2P network.

In this doc, you will find information about:

- what a seed node is and how it works
- the duties of a seed node operator
- the benefits of being a seed node operator
- how to get a seed node up and running
- where to get in touch
```

Reply:
aggreed and done